src/kedro_vizro_labeling/dashboards/protected_action.py

```diff
@@ -32,6 +32,8 @@ class ProtectedAction(vm.Action):
     # TODO: Commented out because could not add_type properly otherwise. type is now "action"
     # type: Literal["protected_action"] = "protected_action"
 
+    # AM comment: some of these are probably constant for the whole app I guess?
+    # I would guess only groups, check_type would be different for different instances of ProtectedAction?
```

Comment:
I believe `group_key` and `group_str_split` are indeed constant for the whole app.